composer.json

```diff
@@ -4,7 +4,7 @@
     "license": "GPL-3.0-only",
     "require": {
         "openspout/openspout": "^3.4 || ^4",
-        "discoverygarden/dgi_migrate": "^1 || ^2 || ^3",
+        "discoverygarden/dgi_migrate": "^1 || ^2 || ^3.16",
```

Comment:
Should be updated according to whatever version https://github.com/discoverygarden/dgi_migrate/pull/162 creates.

Reply:
PR created dgi_migrate tag v3.16.0, so this should be good to leave as is.